package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "leader-board",
+  "version": "1.0.0",
+  "description": "The leader board is a small website system built to dispaly players score according to the high rate",
+  "private": "true",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open ",
+    "build": "webpack"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/joskalenda/leader-board.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/joskalenda/leader-board/issues"
+  },
+  "homepage": "https://github.com/joskalenda/leader-board#readme",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "css-loader": "^6.5.1",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.25.4",
+    "hint": "^6.1.9",
+    "html-webpack-plugin": "^5.5.0",
+    "style-loader": "^3.3.1",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
```

Comment:
Hello Sammit, Thanks for the review you provided, for the change you have suggested, I think I have webpack installed already
I am running `npm start` on my local computer and it works because I installed webpack already and pushed it too, as you can see in the image below, it figures in the `dev dependencies`, I pushed it on the `Install webpack config packages` commit on line `33, 34`.
Would you please try to run `npm install` for installing dependencies globally and see if it will be fixed?
<img width="1280" alt="Screen Shot 2022-01-26 at 20 18 02" src="https://user-images.githubusercontent.com/77843050/151222841-ca41af7d-d0f1-442e-b691-91df59d7601a.png">

Reply:
Hi @joskalenda, I already try this in the previous review and the project gives me the error. For you, the project is working because you installed it locally but when someone tries to install the dependencies in their local machine `webpack` is not installed because you don't have `webpack` in the dependencies of your project. You have `webpack-cli` and `webapck-dev-server` but you don't have `webpack`. 

Just add ` "webpack": "^5.38.1" ` to the devDependencies and this will fix the problem. 

Best regards.

Reply:
Alright, thanks for the clarification, now I get the point.
am going to do it.